test/XRP/xrp-client-integration.test.ts

```diff
@@ -71,36 +73,39 @@ describe('XRPClient Integration Tests', function (): void {
   > {
     this.timeout(timeoutMs)
 
-    const doesExist = await xrpWebClient.accountExists(recipientAddress)
-    assert.equal(doesExist, true)
+    const doesAccountExist = await xrpWebClient.accountExists(recipientAddress)
+    assert.equal(doesAccountExist, true)
   })
 
   it('Check if Account Exists - true - rippled', async function (): Promise<
     void
   > {
     this.timeout(timeoutMs)
 
-    const doesExist = await xrpClient.accountExists(recipientAddress)
+    const doesAccountExist = await xrpClient.accountExists(recipientAddress)
 
-    assert.equal(doesExist, true)
+    assert.equal(doesAccountExist, true)
   })
 
   it('Check if Account Exists - false - rippled', async function (): Promise<
     void
   > {
     this.timeout(timeoutMs)
 
-    const coinbaseMainnet = 'XVYUQ3SdUcVnaTNVanDYo1NamrUukPUPeoGMnmvkEExbtrj' // valid address but should NOT show up on testnet
-    const doesExist = await xrpClient.accountExists(coinbaseMainnet)
-    assert.equal(doesExist, false)
+    // A valid address but should NOT show up on testnet
+    const coinbaseMainnet = 'XVYUQ3SdUcVnaTNVanDYo1NamrUukPUPeoGMnmvkEExbtrj'
+
+    const doesAccountExist = await xrpClient.accountExists(coinbaseMainnet)
+
+    assert.equal(doesAccountExist, false)
   })
 
   it('Payment History - rippled', async function (): Promise<void> {
     this.timeout(timeoutMs)
 
     const payments = await xrpClient.paymentHistory(recipientAddress)
 
-    assert.exists(payments && payments.length > 0)
+    assert.isTrue(payments.length > 0)
```

Comment:
I changed the assertion here because `assert.exists()` makes sure it's not `null` or `undefined`, which would be true if `payments.length` was true _or_ false, which felt odd.

Also, I'm not sure that this is _true_, but ESLint + TypeScript were telling me it was impossible for the resulting payments array to not exist based on the typing of `xrpClient.paymentHistory()`, so I removed the `payments &&`.